demos/text_detection_demo/src/text_detection.cpp

```diff
@@ -100,6 +100,34 @@ std::vector<cv::RotatedRect> maskToBoxes(const cv::Mat &mask, float min_area, fl
     return bboxes;
   }
 
+std::vector<cv::RotatedRect> coordToBoxes(const std::vector<float> &coords, float min_area, float min_height,
+                                         cv::Size image_size) {
+    std::vector<cv::RotatedRect> bboxes;
+    int max_bbox_idx = coords.size() / 5;
+    float x_scale = image_size.width / 704.0;
```

Comment:
What does `704.0` magic number mean?

Reply:
It is input_shape for the model, so every picture is resized to (704, 704). I use x_scale and y_scale to resize predicted bounding boxes to the initial picture resolution.